assets/hydrate.sh

```diff
@@ -67,7 +67,8 @@ sudo php -f "${PS_FOLDER}/install/index_cli.php" -- \
   --all_languages=0 \
   --newsletter=0 \
   --send_email=0 \
-  --ssl=0
+  --ssl=0 \
+  "$([ "$PRE_INSTALLED_MODULES" == "true" ] && echo "--modules=$(find "$PS_FOLDER"/modules/* -maxdepth 0 -type d -exec basename {} \; | paste -s -d ',')" || echo "" )"
```

Comment:
Should be the default behaviour: `--modules=$(find "$PS_FOLDER"/modules/* -maxdepth 0 -type d -exec basename {} \; | paste -s -d ',')` shall be run all times at build time.
Do not know any user that would not required it for now.

Reply:
This is a bug fix here.
Should be tested with 1.6, 1.7, 8 & nightly. 

`PRE_INSTALLED_MODULES` is not required, as this should be the default.